src/cpp/core/system/Xdg.cpp

```diff
@@ -223,46 +223,35 @@ void verifyUserDirs(
    const boost::optional<FilePath>& homeDir)
 {
 #ifndef _WIN32
-   auto testDir = [](const FilePath& dir, const ErrorLocation& errorLoc)
+   auto testDir = [](const FilePath& dir)
```

Comment:
I realize it wasn't being used before, but do we not want to pass the external ERROR_LOCATION as it arguably gives more meaningful error messages (i.e. what was trying to test the directory)?

Reply:
That's a good point! I plumbed it back through.